tests/conftest.py

```diff
@@ -69,6 +71,12 @@ def skipif(items, whole_module=False):
            isinstance(configuration['platform'], Cpu64):
             skipit = "`icc+cpu64` won't work with this test"
             break
+        # Skip if it won't run with OneAPICompiler
+        if i == 'cpu64-icpx' and \
```

Comment:
I don't think we need that, if everything is a proper `IntelCompiler` then we just need `cpu64-intel`

Reply:
Can do that, but we have tests that, at least temporarily, pass with ICC but not with ICX.
What you propose should be fine with the next release and at some point, we are gonna drop ICC entirely.
I do not have any string preference